9.secretstore/README.md

```diff
@@ -0,0 +1,148 @@
+- [Postman](https://www.getpostman.com/) [Optional]
+
+Also, unless you have already done so, clone the repository with the samples and ````cd```` into the right directory:
+```
+git clone https://github.com/dapr/samples.git
+cd samples
+```
+  
+## Step 1 - Setup Dapr on your Kubernetes Cluster
+
+The first thing you need is an RBAC enabled Kubernetes cluster. This could be running on your machine using Minikube, or it could be a fully-fledged cluser in Azure using [AKS](https://azure.microsoft.com/en-us/services/kubernetes-service/). 
+
+Once you have a cluster, follow the steps below to deploy Dapr to it. For more details, look [here](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#installing-dapr-on-a-kubernetes-cluster)
+
+> Please note, that using the CLI does not support non-default namespaces.  
+> If you need a non-default namespace, Helm has to be used (see below).
+
+```
+$ dapr init --kubernetes
+ℹ️  Note: this installation is recommended for testing purposes. For production environments, please use Helm
+
+⌛  Making the jump to hyperspace...
+✅  Deploying the Dapr Operator to your cluster...
+✅  Success! Dapr has been installed. To verify, run 'kubectl get pods -w' in your terminal
+```
+
+## Step 2 - Configure a Secret in the Secret Store
+
+Dapr can use a number of different secret stores (AWS Secret Manager, Azure Key Vault, GCP Secret Manager, Kubernetes, etc) to retrieve secrets. For this demo, we'll use Kubernetes. For other secret stores, you can follow the instructions at https://github.com/dapr/docs/tree/master/howto/setup-secret-store
+
+1. Add your secret to a file ./secret. For example, if your password is "abcd", then the contents of ./secret should be "abcd"
+2. Create the secret in the Kubernetes secret store. Note, the name of our secret here is "mysecret" and will be used in a later step.
+    ```
+    kubectl create secret generic mysecret --from-file ./secret
+    ```
+3. You can check that the secret is indeed stored in the Kubernetes secret store by running the command:
+    ```
+    kubectl get secret mysecret -o yaml
+    ```
+   You can see the output as below where the secret is stored in the secret store in Base64 encoded format
+   ```
+    % kubectl get secret mysecret -o yaml
+        apiVersion: v1
+        data:
+        secret: eHl6OTg3Ngo=
+        kind: Secret
+        metadata:
+        creationTimestamp: "2020-05-20T20:20:09Z"
+        name: mysecret
+        namespace: default
+        resourceVersion: "2031800"
+        selfLink: /api/v1/namespaces/default/secrets/mysecret
+        uid: 64c60c3e-dce4-4c02-b5c1-8d4dbb7cbb8f
+        type: Opaque
+    ```
+
+
+## Step 3 - Deploy the Node.js App with the Dapr Sidecar
+
+```
+kubectl apply -f ./deploy/node.yaml
+```
+
+This will deploy our Node.js app to Kubernetes.
+
+You'll also see the container image that we're deploying. If you want to update the code and deploy a new image, see **Next Steps** section. 
```

Comment:
Please avoid using "we" as per https://github.com/dapr/docs/blob/master/contributing/README.md#style-and-tone

Reply:
fixed